packages/ocular/src/services/chat.ts

```diff
@@ -66,9 +66,50 @@ class ChatService extends TransactionBaseService {
     return chat
   }
 
-  async chat(chatId: string, message:string, context: ChatContext): Promise<ChatResponse> {
-    return await this.atomicPhase_(
-      async (transactionManager: EntityManager) => {
+  // async chat(chatId: string, message:string, context: ChatContext): Promise<ChatResponse> {
+  //   return await this.atomicPhase_(
+  //     async (transactionManager: EntityManager) => {
```

Comment:
Do we still need this uncommented code?

Reply:
no we dont need it , just a heads up we need to make all transactrion for the method  , yet to be implemented